pkg/tsdb/cloudwatch/log_sync_query.go

```diff
@@ -82,7 +80,7 @@ var executeSyncLogQuery = func(ctx context.Context, e *cloudWatchExecutor, req *
 }
 
 func (e *cloudWatchExecutor) syncQuery(ctx context.Context, logsClient cloudwatchlogsiface.CloudWatchLogsAPI,
-	queryContext backend.DataQuery, logsQuery models.LogsQuery) (*cloudwatchlogs.GetQueryResultsOutput, error) {
+	queryContext backend.DataQuery, logsQuery models.LogsQuery, logsTimeout models.Duration) (*cloudwatchlogs.GetQueryResultsOutput, error) {
```

Comment:
(Not a blocker at all: I think still I would use the `time.Duration` type here directly and convert time.Duration(instance.Settings.LogsTimeout) or use the time.Duration type of a models.Duration struct in order to avoid this models import.)